### ISSUE TITLE
onChange is not firing

### ISSUE THREAD

**Issue by no12345678** [zenoamaro/react-quill#896]
https://codesandbox.io/p/sandbox/nextjs-dynamic-react-quill-ref-example-forked-v960q5?file=%2Fpages%2Findex.js%3A70%2C25

A little bit messy as i tried so many things..
My issue is the editor doesn't seem to trigger onChange event until I press space or enter..
In so many demos it works fine and the code is excatly the same - I don't understand why this happens, it doesn't make sense.
If anyone has any insight why it might happen, it will help me alot.
Thank you in advance.

**Comment by sachyyn** [comment]
@no12345678 The Sandbox works fine and triggers onChange precisely. You can try in your local environment  once.

**Comment by no12345678** [comment]
> @no12345678 The Sandbox works fine and triggers onChange precisely. You can try in your local environment once.

Yes i know, any reason why it wouldn't work for me locally? it drives me crazy, everything is the same, it doesn't make sense.

**Comment by Toto5931** [comment]
I have got exactly the same problem. On my phone only with capacitor. Any explanation ? Thanks
